aptos-move/framework/aptos-framework/sources/staking_contract.spec.move

```diff
@@ -143,30 +142,30 @@ spec aptos_framework::staking_contract {
 
     /// Account is not frozen and sufficient to withdraw.
     spec create_staking_contract(
-        staker: &signer,
-        operator: address,
-        voter: address,
-        amount: u64,
-        commission_percentage: u64,
-        contract_creation_seed: vector<u8>,
+    staker: &signer,
+    operator: address,
+    voter: address,
+    amount: u64,
+    commission_percentage: u64,
+    contract_creation_seed: vector<u8>,
```

Comment:
something weird

Reply:
[intellij](https://www.google.com/search?sca_esv=abe270d53af57e7a&rlz=1C5GCCM_en&sxsrf=ACQVn0-2uIhyp62gtcxL8CPsV8EeuG9v9w:1711694967004&q=intellij&spell=1&sa=X&ved=2ahUKEwiy8OLT8JiFAxXJpVYBHUN8AuYQkeECKAB6BAgGEAI) move plugin does this automatically and I have no way to disable it...